base_layer/wallet_ffi/src/lib.rs

```diff
@@ -4821,6 +4877,12 @@ mod test {
 
             completed_transaction_destroy(cancelled_tx);
 
+            let split_msg = CString::new("Test Coin Split").unwrap();
+            let split_msg_str: *const c_char = CString::into_raw(split_msg) as *const c_char;
+            wallet_coin_split(alice_wallet, 1000, 3, 25, split_msg_str, 0, error_ptr);
```

Comment:
It occured to me we need the TxId because I would say to properly test this function you should query the wallet after this call to see that the transaction does indeed exist. I wouldn't query using the FFI functions just directly use the Tx_service handle because you just want to see that the completed tx is added.